sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -186,7 +205,29 @@ export class EducationProgramController {
       approvalStatus: educationProgram.approvalStatus,
       programIntensity: educationProgram.programIntensity,
       institutionProgramCode: educationProgram.institutionProgramCode,
+      institutionId: educationProgram.institution.id,
+      institutionName: educationProgram.institution.legalOperatingName,
+      submittedOn: educationProgram.submittedOn,
+      submittedByFirstName: educationProgram.submittedBy?.firstName,
+      submittedLastName: educationProgram.submittedBy?.lastName,
+      effectiveEndDate: educationProgram.effectiveEndDate,
     };
+
+    if (educationProgram.approvalStatus === ApprovalStatus.denied) {
```

Comment:
Is this logic something that really needs be on the API side?
Just looking at this part it seems that it not an API concern deal with it and the API should just return the data.